derive/src/encode.rs

```diff
@@ -14,14 +14,12 @@
 
 use std::str::from_utf8;
 
-use proc_macro2::{Span, TokenStream};
-use syn::{
-	Data, Field, Fields, Ident,
-	punctuated::Punctuated,
-	spanned::Spanned,
-	token::Comma,
-	Error,
-};
+use proc_macro2::{Ident, Span, TokenStream};
+use syn::punctuated::Punctuated;
+use syn::spanned::Spanned;
+use syn::token::Comma;
+use syn::{Data, Field, Fields, Error};
```

Comment:
Actually I think the grouped way is preferred. at least it seems to be the pattern used in substrate